### DISCUSSION TITLE
How can clear   all content  of log  file

### DISCUSSION THREAD

**Discussion by samantrader** [megahertz/electron-log#321] [Q&A]
hi   
How can clear   all content of  log  file  with  code  ?

**Comment by megahertz** [comment]
https://github.com/megahertz/electron-log/blob/master/docs/transports/file.md#getfilemessage-partiallogmessage--logfile

**Reply by danielzhang5566** [reply]
hello, the link is "404" now, please update, thank u

**Reply by megahertz** [reply]
New docs don't contain `clear` description. It's actually `transports.file.getFile().clear()`
